all-unix-style.yml

```diff
@@ -150,6 +150,22 @@ properties:
           [the github project](https://github.com/taskcluster/taskcluster-proxy) for more information.
 
           Since: generic-worker 10.6.0
+  onExitStatus:
```

Comment:
It is really unfortunate this is called `onExitStatus` and not `onExitCode` in docker-worker, as it is exit codes it deals with. But better to be consistent with docker-worker, just unfortunate...

Let's keep the name the same, but refer to exit codes in the doc, as that is the usual term.